### PR TITLE
Ubuntu Sans Mono: Version 1.006 added

### PR DESCRIPTION
**Pull request by yanone**
b48a732: [gftools-packager] Ubuntu Sans Mono: Version 1.006 added

* Ubuntu Sans Mono Version 1.006 taken from the upstream repo https://github.com/canonical/Ubuntu-Sans-Mono-fonts at commit https://github.com/canonical/Ubuntu-Sans-Mono-fonts/commit/49ff2713769d07d603d5959999a9939090e8d058.

### CODE REVIEW

**Comment by yanone** [comment]
See https://github.com/google/fonts/pull/7352#issuecomment-1980616363 about ignoring FAILs.

**Comment by emmamarichal** [comment]
Same issue than in Ubuntu Sans

Screenshot 2024-03-07 at 11 43 49

File: ufl/ubuntusansmono/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<p>
+ The
+ <a href="http://font.ubuntu.com">
+  Ubuntu Font Family
+ </a>
+ are a set of matching new libre/open fonts in development during 2010-2011, with further expansion work and bug fixing in 2015 and 2022â€“2023.
```
Comment (emmamarichal) [inline ufl/ubuntusansmono/DESCRIPTION.en_us.html:6]: same as Ubuntu Sans: are a set -> is a set ?
+ 2022â€“2023

File: ufl/ubuntusansmono/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+subsets: "menu"
+axes {
+  tag: "wght"
+  min_value: 400.0
+  max_value: 700.0
+}
+source {
+  repository_url: "https://github.com/canonical/Ubuntu-Sans-Mono-fonts"
+  commit: "49ff2713769d07d603d5959999a9939090e8d058"
+}
```
Comment (emmamarichal) [inline ufl/ubuntusansmono/METADATA.pb:40]: stroke: "SANS_SERIF"
classifications: "MONOSPACE"

**Comment by yanone** [comment]
Updated

**Pull request merged**